components/camel-sjms2/pom.xml

```diff
@@ -66,6 +66,12 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>${awaitility-version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I believe this one is not necessary.

Reply:
Yes, sorry i missed it. Thanks!

Reply:
I have removed the dependency, can you please review it?
Thanks!